### PR TITLE
Adding the execution part in the start stop restart commands

### PR DESCRIPTION
**Pull request by LDuheron**

### CODE REVIEW

File: src/parser/job.rs [hunk 1]
```diff
@@ -84,7 +113,97 @@ impl std::cmp::PartialEq for Job {
...
         println!("log: start {}", job_name);
-        self.is_running = true;
+
+        for i in 0..self.num_procs {
+            let mut command = Command::new(&self.command);
+
+            if let Some(args) = &self.arguments {
+                for element in args.iter() {
+                    command.arg(element);
+                }
```
Comment (tsiguenz) [inline src/parser/job.rs]: use args

File: src/parser/job.rs [hunk 2]
```diff
@@ -84,7 +113,97 @@ impl std::cmp::PartialEq for Job {
...
+            }
+
+            if let Some(child) = self.processes.get_mut(&i) {
+                match child.try_wait() {
+                    Ok(None) => {
+                        println!("Process is already running.");
+                        continue;
```
Comment (tsiguenz) [inline src/parser/job.rs]: do this ealier

Comment (tsiguenz) [reply src/parser/job.rs]: don't need to try wait because if the child exist, the process is running

File: src/parser/job.rs [hunk 3]
```diff
@@ -84,7 +113,97 @@ impl std::cmp::PartialEq for Job {
...
+                    Err(e) => {
+                        eprintln!("Error: {:?}", e);
+                        return;
+                    }
+                }
+            }
+
+            if let Some(environment) = &self.environment {
+                for (key, value) in environment {
+                    command.env(key, value);
```
Comment (tsiguenz) [inline src/parser/job.rs]: can use envs

File: src/parser/job.rs [hunk 4]
```diff
@@ -84,7 +113,97 @@ impl std::cmp::PartialEq for Job {
...
+            //         Err(e) => {
+            //             eprintln!("Error: {:?}", e);
+            //             return;
+            //         }
+            //     }
+            // }
+
+            if let Some(ref work_dir) = self.work_dir {
+                if fs::metadata(work_dir).is_err() {
+                    eprintln!("Error: {:?}", work_dir);
```
Comment (tsiguenz) [inline src/parser/job.rs:149]: https://doc.rust-lang.org/std/path/struct.Path.html#method.is_dir 
use is_dir to check if the path is a directory

**Review by tsiguenz — CHANGES_REQUESTED** [review]
Good job, some things to change but it's pretty :)

**Comment by LDuheron** [comment]
addd comments for the hashmap modifications, check start / stop + modify name of variable

**Review by tsiguenz — APPROVED** [review]

**Pull request merged**